pkg/operator/operator/k8s.go

```diff
@@ -388,6 +388,10 @@ func getEnvVars(api *spec.API, container string) []kcore.EnvVar {
 				Name:  "CORTEX_PROJECT_DIR",
 				Value: path.Join(_emptyDirMountPath, "project"),
 			},
+			kcore.EnvVar{
+				Name:  "S6_BEHAVIOUR_IF_STAGE2_FAILS",
```

Comment:
With this environment variable, is it possible to remove any `if [ $status -ne 0 ]` checks in `bootloader.sh`?